Disable KTLS at build time.

Although an explicit SSL_OP_ENABLE_KTLS is required at runtime
to enable this functionality, using it will mean that TLS
cryptography occurs outwith the FIPS module boundary.

Therefore, if FIPS is wanted, it's safer to disable this at build
time so that there's no opportunity for accidents later.
Index: os-openssl/Configurations/10-main.conf
===================================================================
--- os-openssl.orig/Configurations/10-main.conf
+++ os-openssl/Configurations/10-main.conf
@@ -700,7 +700,7 @@ my %targets = (
         shared_target    => "linux-shared",
         shared_cflag     => "-fPIC",
         shared_ldflag    => sub { $disabled{pinshared} ? () : "-Wl,-znodelete" },
-        enable           => [ "afalgeng", "ktls" ],
+        enable           => [ "afalgeng" ],
     },
     "linux-latomic" => {
         inherit_from     => [ "linux-generic32" ],
